app/models/finance/statement_line_item.rb

```diff
@@ -1,16 +1,18 @@
 # frozen_string_literal: true
 
 class Finance::StatementLineItem < ApplicationRecord
+  self.table_name = "statement_line_items"
+
   BILLABLE_STATES = %w[eligible payable paid].freeze
   REFUNDABLE_STATES = %w[awaiting_clawback clawed_back].freeze
 
-  self.table_name = "statement_line_items"
-
   belongs_to :statement
   belongs_to :participant_declaration
 
+  scope :billable, -> { eligible.or(payable).or(paid) }
```

Comment:
these seem to be duplicates of https://github.com/DFE-Digital/early-careers-framework/pull/2225/files#diff-b12d4a990e9638e969b742c993f93cbdf7fec6dba18934aaa85136a137207c7cL23